aries_cloudagent/resolver/routes.py

```diff
@@ -103,8 +103,8 @@ async def resolve_did(request: web.BaseRequest):
     try:
         session = await context.session()
         resolver = session.inject(DIDResolver)
-        document = await resolver.resolve(context.profile, did)
-        result = document.serialize()
+        resolution: Resolution = await resolver.resolve(context.profile, did)
+        result = resolution.did_doc.serialize()
```

Comment:
We should update the admin API and return the `resolution` (metadata with the doc). 
something like this.
```
result = {
  'did_doc': resolution.did_doc.serialize(),
  'meta_data': resolution. resolver_metadata.serialize()
}
```

Reply:
Hmm, I think it could be interesting if we add a "verbose" or something like that as an input. Good idea!